CHANGELOG.md

```diff
@@ -2,6 +2,33 @@
 
 Mapbox welcomes participation and contributions from everyone.
 
+# 10.0.0-rc.9 September 22, 2021
+
+**The Mapbox Maps SDK for Android has moved to release candidate status and is now ready for production use.**
+
+## Features ✨ and improvements 🏁
+* [docs] Fix documentation for OnMapIdleListener and CameraChangeListeners. ([#645](https://github.com/mapbox/mapbox-maps-android/pull/645))
+* Add support for SymbolZOrder in PointAnnotationManager ([#638](https://github.com/mapbox/mapbox-maps-android/pull/638))
+* Add support for PromoteId to be used with Feature state API. ([#636](https://github.com/mapbox/mapbox-maps-android/pull/636))
+* Expose optimizeForTerrain flag ([#654](https://github.com/mapbox/mapbox-maps-android/pull/654))
+* Enable instant transitions for data driven symbol layer properties.
+
+## Bug fixes 🐞
+* onStyleLoaded/onMapLoaded callbacks are called even if hosting fragment/activity is stopped ([#629](https://github.com/mapbox/mapbox-maps-android/pull/629))
+* Fix drag annotation blink issue ([#639](https://github.com/mapbox/mapbox-maps-android/pull/639))
+* Apply annotation manager properties to drag layer to keep annotations the same while dragging ([#640](https://github.com/mapbox/mapbox-maps-android/pull/640))
+* Fix point annotation update bitmap affect others issue. ([#633](https://github.com/mapbox/mapbox-maps-android/pull/633))
```

Comment:
```suggestion
* Fix point annotation updating all same content bitmaps instead of one particular. ([#633](https://github.com/mapbox/mapbox-maps-android/pull/633))
```